tests/model_fields/test_booleanfield.py

```diff
@@ -26,6 +28,7 @@ def test_booleanfield_get_prep_value(self):
     def test_nullbooleanfield_get_prep_value(self):
         self._test_get_prep_value(models.BooleanField(null=True))
 
+    @ignore_warnings(category=RemovedInDjango40Warning)
```

Comment:
I removed these decorators because we added a system check so they don't ignore anything.